indico/modules/users/schemas.py

```diff
@@ -39,6 +30,17 @@ def add_country_name(self, data, **kwargs):
         return data
 
 
+class UserSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = User
+        fields = ('id', 'identifier', 'first_name', 'last_name', 'email', 'affiliation', 'affiliation_id',
+                  'address', 'title', 'affiliation_meta', 'full_name', 'phone', 'avatar_url')
```

Comment:
does this reveal address/phone in places where we didn't reveal it before (ie users other than yourself)?